packages/calcite-components/src/components/tile/tile.scss

```diff
@@ -7,42 +7,65 @@
  * @prop --calcite-tile-border-color: Specifies the border color of the component.
  * @prop --calcite-tile-description-text-color: Specifies the description text color of the component.
  * @prop --calcite-tile-heading-text-color: Specifies the heading text color of the component.
+ * @prop --calcite-tile-icon-color: Specifies the icon color in the component.
+ * @prop --calcite-tile-selection-icon-color: Specifies the color of the selection icon in the component.
+ * @prop --calcite-tile-selection-icon-color-hover: Specifies the color of the selection icon in the component on hover.
```

Comment:
suggestion: 
replace all three icon tokens and the "tile-description" with 
`--calcite-tile-text-color` and `--calcite-tile-color-selected` which covers the states and retains the overall design pattern

Reply:
How would a user override the icon color separately from the title or description text? Just `--calcite-icon-color?` Wouldn't it be more expected to use `--calcite-tile-icon-color` for the display icon?

Reply:
Because Tile now uses an actual `calcite-icon` for the radio and checkbox UI, the meaning of "icon" easily generates confusion too because there's the "selection" icon (radio or checkbox) and the "content" icon that displays above the heading.  @macandcheese which icon are you referring to here?

Reply:
The content one. I thought the original set you had here made the most sense, just IMO - to your point it makes the distinction between selection icon more clear.

Reply:
Good point. 
Per some other discussions, like a more general "close-button" pattern I think there is a more global "selection-icon" pattern that we could use. As for the tile-icon, what about something like the following?

```css
calcite-icon {
  color: var(--calcite-title-icon-color, var(--calcite-icon-color, currentColor));
}
```

Reply:
Yeah that looks good. I think the “selection-“ prefix does make it kind of clear and we use it elsewhere, so agreed with some consistent naming there it’s not as bad. 

I don’t think using component specific naming is an issue there - even when we have the global overrides I think we’d want publicly documented component-specific ones.


.selection-icon-example-class {
  color: var(--calcite-title-selection-icon-color, var(--calcite-selection-icon-color, “whatever underlying default”));
}

Having these at the component level even if there are underlying higher level props makes documentation easier as well IMO. Users won’t need to track down these possible properties.

Reply:
Agreed!

```css
.selection-icon-example-class {
  color: var(--calcite-title-selection-icon-color, var(--calcite-selection-icon-color, “whatever underlying default”));
}
```

looks good.

Reply:
Am I reading that right?  Should it be --calcite-**tile**-icon-color ?

Reply:
I saw the way Card handles selection styling, so I updated it to match that pattern by defining a `--calcite-tile-selection-color` and `--calcite-tile-selection-color-hover`.  I don't think the word `icon` is necessary in the name for styling the radio/checkbox elements because that leaks an internal implementation detail and just confuses it with the content icon.